### ISSUE TITLE
lack of appropriate aria-roles and aria-current

### ISSUE THREAD

**Issue by nicolethoen** [patternfly/patternfly-react#7171]
- Add aria-label prop to each progress step to documentation
- Add aria-current=step to current progress step
- Aria-current=location for active jump links
- Tabs - use tab role, and tablist role (we were already using tabpanel role)
